include/nitro/broken_options/option/toggle.hpp

```diff
@@ -102,17 +143,51 @@ namespace broken_options
                     // Python code would look like this:
                     // if bool(env_value):
                     // FeelsBadMan
-                    if (env_value == "TRUE" || env_value == "ON" || env_value == "YES" ||
-                        env_value == "true" || env_value == "on" || env_value == "yes" ||
-                        env_value == "1" || env_value == "Y")
+                    if (parse_env_value(env_value))
                     {
                         update_value({ "--" + name() });
                     }
+                    else
+                    {
+                        update_value({ "--no-" + name()});
+                    }
                     return;
                 }
             }
         }
+        bool matches(const argument& arg) override
```

Comment:
Code duplication is something, you should avoid. This method is basically the same as the parent's method plus one additional check. You can refactor this method to call the parent method for the shared conditions and only handle the one special thing here.